docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "2"
+services:
+  apostrophe:
+    container_name: apostrophe
+    restart: always
+    build: .
+    ports:
+      - "3000:3000"
+    links:
+      - mongo
+    volumes:
+      - ./data/uploads:/app/public/uploads
+    environment:
+      - APOS_MONGODB_URI=mongodb://mongo:27017/db
+    depends_on:
+      - mongo
+  mongo:
+    container_name: mongo
+    image: mongo
+    restart: always
+    volumes:
+      - ./data/db:/data/db
+    expose:
+      - "27017"
```

Comment:
If people use this configuration in production, they will expose their db without authentication. Might be useful to explain in the documentation not to expose the port in production or protect the db with credentials

Reply:
Actually not the case, from the Expose docs:

"Expose ports without publishing them to the host machine - they’ll only be accessible to linked services. Only the internal port can be specified."

We could talk about credentials for service-to-service communication, but that moves the goalposts to a greater level of security than we have when typically running apostrophe without containers, so I think it would be overkill for our "starter" docker config.